reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -83,6 +85,7 @@
     private long lastApertureRefresh;
     private long refreshPeriod;
     private volatile long lastRefresh;
+    private final EmptySubject closeNotifier = new EmptySubject();
```

Comment:
It's a little bit confusing to have this member named the same way that the getter.
Why not renaming this one `closeSubject` or something?